tests/phpunit/unit/tests/AMP/Story_Sanitizer.php

```diff
@@ -661,7 +661,7 @@ public function test_use_semantic_heading_tags_existing_headings(): void {
 	<amp-story-page>
 		<amp-story-grid-layer>
 			<p class="text-wrapper" style="font-size:.582524em">Title 1B</p>
-			<h3 class="text-wrapper" style="font-size:.339805em">Title 3B</h3>
+			<p class="text-wrapper" style="font-size:.339805em">Title 3B</p>
```

Comment:
Why do we need to change this test here?

Reply:
With this change to the test, it automatically covers this change in logic in the sanitizer.

It verifies, that this Title 3B element is not changed to a heading because there are other headings in the story.